WalletWasabi.Documentation/Guides/InstallInstructions.md

```diff
@@ -110,58 +113,58 @@ sudo gpg2 --verify {file name}.asc {file name}.dmg
 
 Example: `gpg2 --verify Wasabi-1.1.6.dmg.asc Wasabi-1.1.6.dmg`.
 
-If the message returned says `Good signature from nopara73 aka Ficsór Ádám` and that it was signed with `Primary key fingerprint: 21D7 CA45 565D BCCE BE45  115D B4B7 2266 C47E 075E`, then the software was not tampered with since the developer signed it.
+If the message returned says `Good signature from nopara73 aka Ficsór Ádám` and that it was signed with Primary key fingerprint: `21D7 CA45 565D BCCE BE45  115D B4B7 2266 C47E 075E`, then the software was not tampered with since the developer signed it.
```

Comment:
Nit: 
Imo, this should be left as it is, because `Primary key fingerprint: 21D7 CA45 565D BCCE BE45  115D B4B7 2266 C47E 075E` is all part of the quoted output message (it also wouldn't make sense to capitalize `primary` otherwise).